src/regions/RectRegion.js

```diff
@@ -15,6 +15,11 @@ import { LabelOnRect } from "../components/ImageView/LabelOnRegion";
 import { guidGenerator } from "../core/Helpers";
 import { AreaMixin } from "../mixins/AreaMixin";
 import { getBoundingBoxAfterChanges, fixRectToFit } from "../utils/image";
+import chroma from 'chroma-js';
+import { useRegionColors } from "../hooks/useRegionColor";
+import { AliveRegion } from "./AliveRegion";
+
+chroma;
```

Comment:
missed?

Reply:
nah, was just observing the api autocompletion and forgot to remove it

Reply:
nah, was just observing api autocompletion and forgot to remove it